incubator/spark-history-server/.helmignore

```diff
@@ -0,0 +1,14 @@
+# Patterns to ignore when building packages.
+# This supports shell glob matching, relative path matching, and
+# negation (prefixed with !). Only one pattern per line.
+
+.DS_Store
+.git/
+.gitignore
+*.tmp
+*~
+
+# Various IDEs
+.project
+.idea/
+*.tmproj
```

Comment:
There should be more listed in the `.helmignore` file. For example, see https://github.com/helm/charts/blob/master/stable/drone/.helmignore

Reply:
Now added the missing files.